extensions/amp-form/0.1/amp-form.js

```diff
@@ -229,12 +237,22 @@ export class AmpForm {
     this.form_.addEventListener(
         'submit', this.handleSubmitEvent_.bind(this), true);
     this.form_.addEventListener('blur', e => {
-      onInputInteraction_(e);
+      checkUserValidityAfterInteraction_(dev().assertElement(e.target));
       this.validator_.onBlur(e);
     }, true);
+    this.form_.addEventListener('change', e => {
```

Comment:
Nope, "as-you-go" is sort of poorly named. It should be called something that implies "use your own error message elements instead of using Chrome's validation tooltip, AND do so as the user fills out fields". The `change` listener is still necessary when using verification with Chrome's default validation UI.